{
  "id": "CVE-2020-13254",
  "details": "An issue was discovered in Django 2.2 before 2.2.13 and 3.0 before 3.0.7. In cases where a memcached backend does not perform key validation, passing malformed cache keys could result in a key collision, and potential data leakage.",
  "aliases": [
    "BIT-django-2020-13254",
    "GHSA-wpjr-j57x-wxfw",
    "PYSEC-2020-31"
  ],
  "modified": "2024-05-14T07:34:55.277856Z",
  "published": "2020-06-03T14:15:12Z",
  "related": [
    "DLA-2233-1",
    "DSA-4705-1",
    "USN-4381-1",
    "USN-4381-2"
  ],
  "references": [
    {
      "type": "WEB",
      "url": "https://docs.djangoproject.com/en/3.0/releases/security/"
    },
    {
      "type": "ADVISORY",
      "url": "https://security.netapp.com/advisory/ntap-20200611-0002/"
    },
    {
      "type": "ADVISORY",
      "url": "https://www.debian.org/security/2020/dsa-4705"
    },
    {
      "type": "ARTICLE",
      "url": "https://www.djangoproject.com/weblog/2020/jun/03/security-releases/"
    },
    {
      "type": "WEB",
      "url": "https://groups.google.com/d/msg/django-announce/pPEmb2ot4Fo/X-SMalYSBAAJ"
    },
    {
      "type": "WEB",
      "url": "https://lists.debian.org/debian-lts-announce/2020/06/msg00016.html"
    },
    {
      "type": "WEB",
      "url": "https://www.oracle.com/security-alerts/cpujan2021.html"
    },
    {
      "type": "WEB",
      "url": "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/4A2AP4T7RKPBCLTI2NNQG3T6MINDUUMZ/"
    },
    {
      "type": "WEB",
      "url": "https://usn.ubuntu.com/4381-1/"
    },
    {
      "type": "WEB",
      "url": "https://usn.ubuntu.com/4381-2/"
    }
  ],
  "affected": [
    {
      "ranges": [
        {
          "type": "GIT",
          "repo": "https://github.com/django/django",
          "events": [
            {
              "introduced": "635d53a86a36cde7866b9caefeb64d809e6bfcd9"
            },
            {
              "introduced": "d36413a2eb5678dda35559a48769703acd784898"
            },
            {
              "fixed": "8093aaa8ff9dd7386a069c6eb49fcc1c5980c033"
            }
          ]
        }
      ],
      "versions": [
        "2.2",
        "2.2.1",
        "2.2.10",
        "2.2.11",
        "2.2.12",
        "2.2.2",
        "2.2.3",
        "2.2.4",
        "2.2.5",
        "2.2.6",
        "2.2.7",
        "2.2.8",
        "2.2.9"
      ],
      "database_specific": {
        "source": "https://storage.googleapis.com/cve-osv-conversion/osv-output/CVE-2020-13254.json"
      }
    }
  ],
  "schema_version": "1.6.0",
  "severity": [
    {
      "type": "CVSS_V3",
      "score": "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:U/C:H/I:N/A:N"
    }
  ]
}